### ISSUE TITLE
EG.5.1.1 Sublineage with S:L455F (3 seq, 3 provinces, China; July 15)

### ISSUE THREAD

**Issue by ryhisner** [sars-cov-2-variants/lineage-proposals#414]
**Description**
**Sub-lineage of:** EG.5.1.1 (XBB.1.9.2.5.1.1)
**Earliest sequence:** 2023-6-20, China, Henan, Puyang — EPI_ISL_17988152
**Most recent sequence:** 2023-7-5, China, Hubei, Xiangyang — EPI_ISL_17988079
**Countries circulating:** China (3)
**Number of Sequences:** 3
**GISAID AA Query:** Spike_L455F, Spike_F456L, NSP12_D63N, Spike_F486P
**GISAID Nucleotide Query:** C6541T, G13627A, G22927T, C25572T
**CovSpectrum Query:** Nextcladepangolineage: EG.5.1.1* & S:L455F
**Substitutions on top of EG.5.1.1:**
**Spike:** L455F
**Nucleotide:** G22927T

**USHER Tree**
https://nextstrain.org/fetch/raw.githubusercontent.com/ryhisner/jsons/main/EG.5.1.1%20+%20L455F.json?c=gt-S_455&label=id:node_5182912
<img width="1489" alt="image" src="https://github.com/sars-cov-2-variants/lineage-proposals/assets/33738461/72045743-ebc0-4bf3-b3a6-d4e49a32b13e">

**Evidence**
Self-explanatory. There's a fourth sequence of EG.5.1.1 with S:L455F, but it appears to be unrelated to the other three. These are all from separate provinces and are separated from one another by at least four nucleotide differences, so this lineage may well be fairly widespread. Too early to tell for sure.

 
**Genomes**
<details>
<summary>Genomes</summary>
EPI_ISL_17987533, EPI_ISL_17988079, EPI_ISL_17988152
</details>

**Comment by FedeGueli** [comment]
To be proposed onxe get 10seqs

**Comment by HynnSpylor** [comment]
Ah I'm just planning to propose it!
S:L455F+S:F456L "exchange" is popular recently (in XBB.1.5.59). Nowadays EG.5.1* has risen rapidly, and it has the highest proportion in China since mid June. It deserves a further watch.

**Comment by HynnSpylor** [comment]
one more seq from Sichuan Province, China

**Comment by BorisUitham** [comment]
There's also an eg.5.1.1 with 455f from shanghai uploaded on 07-18. EPI_ISL_17994488, but it seems to be on another branch than this one, with C11779T

**Comment by aviczhl2** [comment]
Designated HK.3

**Comment by FedeGueli** [comment]
https://github.com/cov-lineages/pango-designation/commit/39a35ba57d785e6bdda763fd612b9d7ba4442119

**Comment by oobb45729** [comment]
S:L455F looks very homoplasic within the EG.5.1 tree now. The supposed lineages is defined by a single mutation. I'm not convinced. They might gain S:L455F independently. @corneliusroemer

**Comment by ryhisner** [comment]
Yeah, you have to look closely to make sure any S:L455F lineages are monophyletic. In this case, I think you have to assume it is until the tree fills out further.

**Comment by BorisUitham** [comment]
@oobb45729 there are definitely a few convergemt branches (i believe about 6-7 within eg.5*)  with 455f: one in eg2, his eg.5.1.1 designated as hk.3, etc. But there are a lot of interesting singlets. Perhaps this should be monitored over these weeks to see if the singlets might still cause further spread.
